addons/web/static/src/legacy/js/core/misc.js

```diff
@@ -101,9 +100,6 @@ function unblockUI() {
  * If wait is true, sleep 1s and wait for the server i.e. after a restart.
  */
 function redirect (url, wait) {
-    // Dont display a dialog if some xmlhttprequest are in progress
-    disableCrashManager();
```

Comment:
not necessary anymore?

Reply:
Check what happens if redirected while other RPCs are ongoing